### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by Bikaze**
Update a line in README.md

### CODE REVIEW

**Review by Bikazemcclent — COMMENTED** [review]
Is there some way to improve this?

**Comment by Bikaze** [comment]
yes

**Review by Bikazemcclent — CHANGES_REQUESTED** [review]
change line 1 and 2

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# CodoGex
+# Group 16
```
Comment (Bikazemcclent) [inline README.md:1]: why this line?

**Review by Bikazemcclent — CHANGES_REQUESTED** [review]

**Review by Bikazemcclent — APPROVED** [review]
Looks good!

**Pull request merged**